### DISCUSSION TITLE
Support for Ruby runtime 3.x.x in Lambda ?

### DISCUSSION THREAD

**Discussion by ghost** [aws/aws-sdk-ruby#2721] [General]
Hello, 

Does anyone know if the support for Ruby 3.1.1 is in the pipeline or planned ahead ?

Thank you,
Sriram

**Comment by mullermp** [comment]
I assume you mean for Lambda runtime. We don't control that on the SDK team. I'll forward this to the Lambda team.

**Reply by ghost** [reply]
Thank you @mullermp
